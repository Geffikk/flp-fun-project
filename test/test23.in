X,C,S,A
a
S
X->aa
X->C
X->S
C->AAA
C->a
S->aA
S->AX
