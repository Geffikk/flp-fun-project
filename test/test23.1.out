X,C,S,A
a
S
X->aa
X->AAA
X->a
X->aA
X->AX
C->AAA
C->a
S->aA
S->AX
